docs/manual/setup/getting-started.md

```diff
@@ -0,0 +1,183 @@
+process our observability data.
+
+Here's a script to install Vector:
+
+```shell
+curl --proto '=https' --tlsv1.2 -sSf https://sh.vector.dev | sh
+```
+
+Or you can [choose your preferred installation method][docs.installation].
+
+Once we have Vector installed, let's check to make sure that it's working correctly:
+
+```shell
+vector --version
+```
+
+## Configure Vector
+
+Vector runs with a [configuration file][docs.setup.configuration] that tells it what
+[components][pages.components] to run and how they should interact. Vector's components are made up of
+sources, transforms, and sinks:
+
+* [*Sources*][docs.sources] collect or receive data from observability data sources into Vector
+* [*Transforms*][docs.transforms] can manipulate or change that observability data inside Vector
+* [*Sinks*][docs.sinks] send data onwards from Vector to external services or destinations
+
+Let's create a configuration file:
+
+```toml title="vector.toml"
+[sources.foo]
+  type = "stdin"
+
+[sinks.bar]
+  inputs = ["foo"]
+  type = "console"
+  encoding.codec = "text"
+```
+
+Each component has a unique identifier and is prefixed with the type of component, for example, `sources` for a
+source. Our first component, `sources.foo`, uses the [`stdin` source][docs.sources.stdin] which tells Vector to receive
+data over `STDIN` and is identified as `foo`.
+
+Our second component, `sinks.bar`, uses the [`console` source][docs.sinks.console] which tells Vector to simply print
+the data to `STDOUT`. The `encoding.codec` option tells Vector to print the data as plain text (unencoded).
+The `inputs` option creates a pipeline from our source, which receives events, to our sink, which sends those
+events onwards. The `inputs` option in our `sinks.bar` component tells Vector where this sink's events are coming
+from. In our case, events are received from our other component, the source identified as `foo`.
+
+## Hello World!
+
+That's it for our first config, now; let's pipe an event through it:
+
+```bash
+echo 'Hello World!' | vector --config ./vector.toml
+```
+
+The `echo` statement sends a single log to Vector over `STDIN`. The `vector...` command starts Vector with our
+previously created config file.
+
+The event we've just sent will get received by our `sources.foo` component, then sent onto the `sinks.bar` component,
+which will, in turn, echo it back to the console:
+
+```text
+... some logs ...
+Hello World!
+```
+
+> Hey, if you want to see something cool, try setting `encoding.codec = "json"` in the sink config.
+
+## Hello World Mark II
+
+The echoing of events isn't very exciting, though. Let's see what we can do with some real observability data.
+Let's take a classic problem, collecting and processing Syslog events, and see how Vector handles it.
+
+To do this, we're going to add two new components to our configuration file. The first component uses
```

Comment:
Here, I would move the configuration up and follow it with the explanation. I tend to find that more engaging than reading a few paragraphs about what you're *going to see* before seeing it